src/main/java/org/embulk/input/zendesk/clients/ZendeskRestClient.java

```diff
@@ -127,27 +126,34 @@ public void onGiveup(Exception firstException, Exception lastException)
     @VisibleForTesting
     protected HttpClient createHttpClient()
     {
-        RequestConfig config = RequestConfig.custom()
+        final RequestConfig config = RequestConfig.custom()
                 .setConnectTimeout(CONNECTION_TIME_OUT)
                 .setConnectionRequestTimeout(CONNECTION_TIME_OUT)
                 .build();
         return HttpClientBuilder.create().setDefaultRequestConfig(config).build();
     }
 
-    private String sendGetRequest(final String url, final PluginTask task) throws ZendeskException
+    private String sendGetRequest(final String url, final PluginTask task)
+            throws ZendeskException
     {
         try {
-            HttpClient client = createHttpClient();
-            HttpRequestBase request = createGetRequest(url, task);
+            final HttpClient client = createHttpClient();
+            final HttpRequestBase request = createGetRequest(url, task);
+
+            if (rateLimiter != null) {
+                rateLimiter.acquire();
+            }
             logger.info(">>> {}{}", request.getURI().getPath(),
                     request.getURI().getQuery() != null ? "?" + request.getURI().getQuery() : "");
-            HttpResponse response = client.execute(request);
-            getRateLimiter(response).acquire();
-            int statusCode = response.getStatusLine().getStatusCode();
+            final HttpResponse response = client.execute(request);
+            if (rateLimiter == null) {
```

Comment:
In case we meet some problem, such as rate limit, at the first time the header will not have **x-rate-limit,** which require to init the rate limit. 
So **rateLimiter** will be null, acquire will throw NPE